### ISSUE TITLE
Unimplemented Operation: fns.FindField

### ISSUE THREAD

**Issue by jbmorley** [inseven/opolua#349]
## Description

OK. I shall finding bugs for the evening. I suspect this might be a database one though.

https://archive.org/download/psioncdvol1/PsionCDVol1.iso/Programs%2F900%2FMetro.SIS

(https://software.psion.info/programs/0x10000e7a/)

## Details

```
Unimplemented operation fns.FindField
METRO\METRO:00000871: 57 [CallFunction] 0x54 (FindField)
stack traceback:
	[C]: in function 'error'
	init.lua:478: in function 'unimplemented'
	fns.lua:894: in function 'fns.FindField'
	ops.lua:1028: in function 'ops.CallFunction'
	runtime.lua:864: in function <runtime.lua:849>
	[C]: in function 'xpcall'
	runtime.lua:983: in method 'pcallProc'
	runtime.lua:1253: in function 'runtime.runOpo'
```

**Comment by tomsci** [comment]
Already fixed in The Great Database Refactor™ Coming Soon to a branch near you

**Comment by tomsci** [comment]
fixed in #350
